app/api/monthly.py

```diff
@@ -6,11 +6,11 @@
 )
```

Comment:
one important this is missing i.e removal of slack module from requuirement.txt .

Reply:
ok sir removing will be done